src/core/load-fragment.c

```diff
@@ -6246,7 +6246,12 @@ int unit_load_fragment(Unit *u) {
                  * verify this here. */
                 f = fopen(fragment, "re");
                 if (!f)
-                        return log_unit_notice_errno(u, errno, "Failed to open %s: %m", fragment);
+                        /* We do not rebuild unit_name_map and friends on transient file being stopped.
```

Comment:
It appears to me that we should fix the underlying issue then?

Reply:
I thought so. But it is heavy and completely waste of time to rebuild the tables after each transient unit stopped.

Of course, it may be possible to remove the relevant entries in the tables, but not sure it is reliable. On successful path, we can do. But I am not sure for all failure paths.

Reply:
I will try tomorrow again.

Reply:
Yeah, I'm aware that the unit cache logic needs more love (c.f. https://github.com/systemd/systemd/pull/35711). I'd still prefer not to tape over the problem, especially that it's just a log msg and not really new.